exercises/pangram/canonical-data.json

```diff
@@ -1,97 +1,90 @@
-        },
-        {
-          "description": "recognizes a perfect lower case pangram",
-          "property": "isPangram",
-          "input": {
-            "sentence": "abcdefghijklmnopqrstuvwxyz"
-          },
-          "expected": true
-        },
-        {
-          "description": "pangram with only lower case",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the quick brown fox jumps over the lazy dog"
-          },
-          "expected": true
-        },
-        {
-          "description": "missing character 'x'",
-          "property": "isPangram",
-          "input": {
-            "sentence": "a quick movement of the enemy will jeopardize five gunboats"
-          },
-          "expected": false
-        },
-        {
-          "description": "another missing character, e.g. 'h'",
-          "property": "isPangram",
-          "input": {
-            "sentence": "five boxing wizards jump quickly at it"
-          },
-          "expected": false
-        },
-        {
-          "description": "pangram with underscores",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the_quick_brown_fox_jumps_over_the_lazy_dog"
-          },
-          "expected": true
-        },
-        {
-          "description": "pangram with numbers",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the 1 quick brown fox jumps over the 2 lazy dogs"
-          },
-          "expected": true
-        },
-        {
-          "description": "missing letters replaced by numbers",
-          "property": "isPangram",
-          "input": {
-            "sentence": "7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog"
-          },
-          "expected": false
-        },
-        {
-          "description": "pangram with mixed case and punctuation",
-          "property": "isPangram",
-          "input": {
-            "sentence": "\"Five quacking Zephyrs jolt my wax bed.\""
-          },
-          "expected": true
-        },
-        {
-          "description": "upper and lower case versions of the same character should not be counted separately",
-          "property": "isPangram",
-          "input": {
-            "sentence": "the quick brown fox jumps over with lazy FX"
-          },
-          "expected": false
-        }
-      ]
+      "description": "empty sentence",
+      "property": "isPangram",
+      "input": {
+        "sentence": ""
+      },
+      "expected": false
+    },
+    {
+      "description": "perfect lower case",
+      "property": "isPangram",
+      "input": {
+        "sentence": "abcdefghijklmnopqrstuvwxyz"
+      },
+      "expected": true
+    },
+    {
+      "description": "only lower case",
+      "property": "isPangram",
+      "input": {
+        "sentence": "the quick brown fox jumps over the lazy dog"
+      },
+      "expected": true
+    },
+    {
+      "description": "missing character 'x'",
```

Comment:
```suggestion
      "description": "missing an 'x'",
```
or perhaps
```suggestion
      "description": "lacks an 'x'",
```

Similar pattern could be repeated if others are on board.  Also, the whole file essentially appears to have been altered.  Some may find it more difficult to review.

Reply:
I think "lacks" might be harder for non-native-english speakers to grok.

Reply:
Perhaps `"Missing the letter 'x'"`

Reply:
@bencoman `missingTheLetterX` or `missingAnX`?  I agree `lacksAnX` is the least desirable choice.

Reply:
In general I think scaffold words like "the" and "an" can often be left out without compromising meaning - but I can slide on it.  The majority of improvement from this PR comes in removing the nesting platitude description "Check if the given string is a pangram" and shrinking the description "upper and lower case versions of the same character should not be counted separately".  Then once I opened the box, other descriptions get some collateral cleaning (like removing redundant 'panagram' from descriptions.

As for length, my gut feel was 40-60 but I thought it would be good to put some science behind it by reviewing the stock Pharo 7 release...
```smalltalk
classes := Object allSubclasses select: [ :cc | cc isKindOf: TestCase class ].
methods := c flatCollect: [ :c | c allMethods  ].
tests := methods select: [ :m | m selector beginsWith: 'test' ].
lengths := tests collect: [ :m | m selector size ].
lengths asBag keysAndValuesDo: [ :len :count | Transcript crShow: len; show: ','; show: count  ]
```
and graphing in Excel...
![image](https://user-images.githubusercontent.com/1713447/59859712-dabead80-93af-11e9-9955-7f1b76653d71.png)

So 40-60 looks like a reasonable target to aim for, but its not an absolute. I arbitrarily picked fixing descriptions longer than 100 chars.  When I get there I'll be able to better evaluate if more is needed. (and it looks like I should review of Pharo's longer ones also)